### PR TITLE
Update Post

### PR DESCRIPTION
**Pull request by jacobhausler**
Update Post works! I also had to re-fix session id? idk.

### CODE REVIEW

File: src/modules/post/mutations.js [hunk 1]
```diff
@@ -58,3 +59,49 @@ export const Delete = types => ({
...
+
+    // will fail if the post doesn't exist
+    const oldPost = await db.findBy('posts', { id: strippedId });
+
+    assert(currentUserID === oldPost.authoredByUserId, "That's not your post!");
+
+    const changeFields = chain(input).
+      omit('id').
+      omit('clientMutationId').
+      omitBy('isUndefined').
```
Comment (slightlytyler) [inline src/modules/post/mutations.js]: this won't work, we need to use the `isUndefined` function provided by lodash

File: src/modules/user/mutations.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import assert from 'assert';
 import bcrypt from 'bcrypt-as-promised';
-import uuid from 'uuid';
+import uuid from 'uuid-js';
```
Comment (slightlytyler) [inline src/modules/user/mutations.js]: Also have `uuid` installed, think that's my bad. need to use only a single library

File: src/modules/user/mutations.js [hunk 3]
```diff
@@ -37,7 +37,9 @@ export const SignIn = types => ({
       await bcrypt.compare(password, user.passwordHash);
 
       session.currentUserID = user.id;
-      session.id = uuid.v1().hex;
+      session.id = uuid.create().hex;
+
+      console.log(session.id);
```
Comment (slightlytyler) [inline src/modules/user/mutations.js]: I removed the linting rule for console logs but we should remove extraneous "development" logs

File: src/modules/post/mutations.js [hunk 4]
```diff
@@ -58,3 +59,49 @@ export const Delete = types => ({
...
+    const { db, session } = context;
+    const { currentUserID } = session;
+    const strippedId = id.split(':')[1];
+
+    // will fail if the post doesn't exist
+    const oldPost = await db.findBy('posts', { id: strippedId });
+
+    assert(currentUserID === oldPost.authoredByUserId, "That's not your post!");
+
+    const changeFields = chain(input).
```
Comment (slightlytyler) [inline src/modules/post/mutations.js]: Guess linter doesn't catch this but the dot operator should start a row, not end it

File: src/modules/post/mutations.js (hunk 1 above)
Comment (jacobhausler) [inline src/modules/post/mutations.js]: done and tested in new commit

Comment (slightlytyler) [inline src/modules/post/mutations.js]: @jacobhausler don't have to comment before you commit, if you change the line it'll hide the comments

**Comment by jacobhausler** [comment]
Latest c1e45b4 should address all of the notes

**Comment by slightlytyler** [comment]
LGTM

**Pull request merged**